RLTest/redis_std.py

```diff
@@ -241,11 +243,16 @@ def createCmdArgs(self, role):
             cmdArgs += ['--tls-replication', 'yes']
 
         if self._getRedisVersion() > 70000:
-            if self.enableDebugCommand:
-                cmdArgs += ['--enable-debug-command', 'yes']
-            if self.enableProtectedConfigs:
-                cmdArgs += ['--enable-protected-configs', 'yes']
-
+            if self.allowUnsafe:
```

Comment:
I believe that if `allow unsafe` option will just enable the other three options then this args duplication will not be required.